rest-api-spec/src/main/resources/rest-api-spec/api/indices.exists.json

```diff
@@ -13,14 +13,6 @@
         }
       },
       "params": {
-        "ignore_unavailable": {
-            "type" : "boolean",
-            "description" : "Whether specified concrete indices should be ignored when unavailable (missing or closed)"
-        },
-        "allow_no_indices": {
-            "type" : "boolean",
-            "description" : "Whether to ignore if a wildcard indices expression resolves into no concrete indices. (This includes `_all` string or when no indices have been specified)"
-        },
```

Comment:
I think we should document this a breaking change, just in case users rely on this broken behavior.